src/webcam.md

```diff
@@ -0,0 +1,22 @@
+# Webcam Maintenance
+
+## Set up
+
+Get a laptop and ensure a webcam program is installed (recommended for linux: `cheese`)
+
+Take a photo with a working webcam as a base to compare others to.
+
+## Procedure
+
```

Comment:
Do you have an estimate of the time taken per camera for this procedure?
No worries if not, though many of our other guides contain this (e.g: https://github.com/srobo/kit-maintenance-docs/blob/master/src/power-board.md#procedure)